packages/vscode-extension/src/devices/preview.ts

```diff
@@ -32,11 +32,15 @@ export class Preview implements Disposable {
         // the promise to prevent the caller from waiting indefinitely
         reject(new Error("Preview server exited without URL"));
       });
+      
+      const streamURLRegex = /(http:\/\/[^ ]*stream\.mjpeg)/;
```

Comment:
Why is it a change here? I believe it is already on main?

Reply:
As above I've merged main into this branch.